### PR TITLE
Change translation on the fly

### PR DESCRIPTION
**Pull request by kepta**
Need to fix the history and map flying to some random area.

cc: @bhousel

### CODE REVIEW

**Comment by bhousel** [comment]
awesome, this will unblock #3755 and #3120

File: modules/ui/init.js [hunk 1]
```diff
@@ -228,6 +228,7 @@ export function uiInit(context) {
 
         var mapDimensions = map.dimensions();
 
+
```
Comment (pnorman) [inline modules/ui/init.js]: extra newline

File: modules/ui/init.js [hunk 2]
```diff
@@ -247,6 +248,7 @@ export function uiInit(context) {
             };
         }
 
+
```
Comment (pnorman) [inline modules/ui/init.js]: extra newline

**Comment by bhousel** [comment]
> How can I test this?

I'm pretty close to merging it, but you can check out this branch and when running, open a developer console call `restart` with a language code:

* `restart('ru')`
* `restart('fr')`
* `restart('ar')`

etc...

File: modules/ui/init.js (hunk 1 above)
Comment (bhousel) [inline modules/ui/init.js]: fwiw, I've been adding 2 spaces before function blocks and anywhere else I want documentation to go.

**Pull request merged**